core/exceptions.py

```diff
@@ -3,3 +3,10 @@ class ValidationError(RuntimeError):
 
     def __init__(self, validation_failures):
         self.validation_failures = validation_failures
+
+
+class InitialValidationError(ValidationError):
```

Comment:
Do we want this to be a subclass of `ValidationError`? It would lead to odd behaviour if the two `except` statements were swapped in `ingest`, for example